configure.ac

```diff
@@ -4235,16 +4234,11 @@ fi
 AC_SUBST(LIBXML2_LIBS)
 AC_SUBST(LIBXML2_CFLAGS)
 
-dnl ### Use libvterm (-lvterm) if available
-HAVE_LIBVTERM=no
-if test "${with_libvterm}" != "no"; then
-  EMACS_CHECK_MODULES([LIBVTERM], [vterm])
-  if test "${HAVE_LIBVTERM}" = "yes"; then
-    AC_DEFINE(HAVE_LIBVTERM, 1, [Define to 1 if libvterm library is present.])
-  fi
+### Use libgit2 and git2rs
+if test "${with_libgit}" = "yes"; then
+  HAVE_LIBGIT=yes
```

Comment:
This shouldn't be necessary right ?

Reply:
You can change the conditional below to be "${with_libgit}" = "yes" if you want to avoid setting HAVE_LIBGIT=yes, but I also think what you have is fine.